### ISSUE TITLE
bitcoinWalletOrImportString

### ISSUE THREAD

**Issue by rbensberg** [decentralized-identity/ion#198]
In the docker readme.md it says that I can confugure as below:
`You can find this in the json file mainnet-bitcoin-docker-config.json file with the parameter, bitcoinWalletOrImportString`
It is not clear to me what to update the bitcoinWalletOrImportString to? Is it just a WIF for a bitcoin address?

**Comment by thehenrytsai** [comment]
Yes, it is a bitcoin WIF string. It is used for writing ION transactions. If you don't plan to write any ION transactions, just to generate a WIF with no BTC in it. There is an outstanding work item to allow this to be left empty for a "reader" mode only node.
